pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1890,6 +1890,8 @@ type StorageScopeSpec struct {
 	Nodes []Node `json:"nodes,omitempty"`
 	// +optional
 	UseAllNodes bool `json:"useAllNodes,omitempty"`
+	// +optional
+	OnlyApplySpecificOSDPlacement bool `json:"onlyApplySpecificOSDPlacement,omitempty"`
```

Comment:
I know I keep changing the name suggestion, but I'm not sure "specific" helps. How about this?
```suggestion
	OnlyApplyOSDPlacement bool `json:"onlyApplyOSDPlacement,omitempty"`
```

Reply:
updated.